src/navigation/components/NavigationItem/components/GithubStars.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import GitHubButton from 'react-github-btn';
+
+import {block} from '../../../../utils';
+
+import {NavigationItemProps} from '../NavigationItem';
+import {NavigationGithubButton} from '../../../../models';
+
+import '../NavigationItem.scss';
+
+const b = block('navigation-item');
```

Comment:
We have component named GithubStar, shoud we name it's className as github-stars?

Reply:
fix, separate styles for sub-components